druid/ingestion-spec/rollups/tpd_index_hourly_syncts.json

```diff
@@ -0,0 +1,176 @@
+{
+  "type": "kafka",
+  "dataSchema": {
+    "dataSource": "tpd-hourly-rollup-syncts",
+    "parser": {
+      "type": "string",
+      "parseSpec": {
+        "format": "json",
+        "flattenSpec": {
+          "useFieldDiscovery": false,
+          "fields": [
+            {
+              "type": "root",
```

Comment:
Please remove the syncts field from the list of fields. It is already included as the timestamp column.